app/infrastructure/persistence/repositories/tenant_integrity_profile_history_repo.py

```
@@ -0,0 +1,76 @@
+        obj = TenantIntegrityProfileHistory(
+            tenant_id=tenant_id,
+            previous_profile=previous_profile,
+            new_profile=new_profile,
+            changed_at=now,
+            changed_by_user_id=changed_by_user_id,
+            change_reason=change_reason,
+            effective_from_seq=effective_from_seq,
+            cooling_off_ends_at=cooling_off_ends_at,
+        )
+        created = await self.create(obj)
+        return created
```

Ruff (0.15.5)

[warning] Unnecessary assignment to `created` before `return` statement

Remove unnecessary assignment

(RET504)